### ISSUE TITLE
Pallete List in Webserver changes nothing

### ISSUE THREAD

**Issue by Guenni75** [jasoncoon/esp8266-fastled-webserver#204]
I don't understand the Palette List (under the Pattern List in the Webserver).
I can choose what ever i want, nothing hapends.

**Comment by henrygab** [comment]
yes, the settings are *very* unclear.  many of them apply only to a specific effect.

There is a small improvement in #211, where at the "section name" for each group of settings is now output in the HTML.  That said, it's still an area of the UI that could be improved.
